### PR TITLE
docs(template):  add setup example with standalone cmp and fix typos

### PR DESCRIPTION
**Pull request by KevTale**
I added an example to the basic setup with standalone components and also fixed several typos or consistency stuff in the `@rx-angular/template` doc.

### CODE REVIEW

File: libs/template/README.md [hunk 1]
```diff
@@ -82,7 +84,23 @@ import { UnpatchModule } from '@rx-angular/template/unpatch';
...
+import { ForModule } from '@rx-angular/template/for';
+import { PushModule } from '@rx-angular/template/push';
+import { UnpatchModule } from '@rx-angular/template/unpatch';
+
+@Component({
+  standalone: true,
+  imports: [ForModule, LetModule, PushModule, UnpatchModule],
+  template: `...`,
+})
+export class AnyComponent {}
```
Comment (edbzn) [inline libs/template/README.md:103]: I would suggest to stick to the standalone approach only, no need to keep the module-based approach IMO.

Note that we removed all the modules, `ForModule` should be replaced with `RxFor`, `LetModule` with `LetDirective`, `PushModule` by `PushPipe`, and `UnpatchModule` with `UnpatchDirective`.

File: apps/docs/docs/template/api/rx-for-directive.md [hunk 2]
```diff
@@ -126,9 +126,9 @@ Standalone component setup:
 ```
 import { ForModule } from "@rx-angular/template/for";
 
-@NgComponent({
+@Component({
     standalone: true,
-    imports: [ ForModule ],
+    imports: [ForModule],
```
Comment (edbzn) [inline apps/docs/docs/template/api/rx-for-directive.md:131]: Here as well, `RxFor` instead.

**Review by edbzn — COMMENTED** [review]
Hey, thanks for the PR! We removed all the modules so we need to remove any reference to them in the doc and replace them with the standalone equivalent.

**Review by edbzn — APPROVED** [review]
Great! Is there any other reference to these modules in the doc?

**Comment by KevTale** [comment]
> Great! Is there any other reference to these modules in the doc?

Good call @edbzn! There was a mention of these modules in cdk/render-strategies doc. I've fixed that and double checked in the entire repo, the only mention left are the actual implementation of the features so I think we are good to go. 👌

**Comment by edbzn** [comment]
Thanks for the contrib @KevTale! :rocket:

**Pull request merged**